src/app/layout/header/header.component.ts

```diff
@@ -54,6 +57,9 @@ export class HeaderComponent implements OnInit {
     _togglz
       .getStateOf('ORCID_ANGULAR_SIGNIN')
       .subscribe(value => (this.togglzOrcidAngularSignin = value))
+    _router.events.subscribe(
+      () => (this.signinRegisterButton = location.path() !== '/signin')
```

Comment:
Let's use the `/singin` from the constants files, in both places

Reply:
@leomendoza123 Done!